Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/UI/Driver.hs

```diff
@@ -989,14 +989,16 @@ validate (personId, _) phoneNumber = do
     let otpHash = smsCfg.credConfig.otpHash
     let altPhoneNumber = phoneNumber.mobileCountryCode <> phoneNumber.alternateNumber
     let sender = smsCfg.sender
+    let smsType = Just Sms.Transactional
```

Comment:
Don't hardcode it here. As currently all our SMS types are transactional, so just hard code these values in shared-kernel